platform/solr/solr-banana/src/main/resources/banana-int/Banana-Default-Dashboard.json

```diff
@@ -0,0 +1,550 @@
+{
+  "title": "Metadata Default Dashboard",
```

Comment:
I updated the default dashboard in the project and the screenshot to show some validation errors.